### PR TITLE
Update readme with content from website

### LINKED ISSUE #943: Add new content and guidelines to Game of Realms readme
## Description

Hey @waymobetta 

Can we add the new website content to the GoR readme? And explain adding a GoR tag to the pull request? 

The worldwide competition to find the best contributors to Gno.land is currently in phase one – for advanced developers only.

Once the necessary tools to start phase two are ready, we’ll open up the competition to newer devs and non-technical contributors.

If you want to stack ATOM rewards and play a key role in the success of Gno.land and web3, read more about Game of Realms or open a [PR](https://github.com/gnolang/gno/) today.

### PR DESCRIPTION
**Pull request by roethke**
Closes: [943](https://github.com/gnolang/gno/issues/943) 

- [x] Add content from [gno.land/game-of-realms](https://gno.land/game-of-realms)
- [x] Add formatting
- [x] Add notice for usage of GoR label (PRs/issues)

### CODE REVIEW

**Comment by roethke** [comment]
Draft until [conversation](https://github.com/gnolang/gno/issues/943) can be resolved.

**Comment by michelleellen** [comment]
@waymobetta how are we able to resolve this?

**Comment by roethke** [comment]
@michelleellen we're just waiting for @gnolang/tech-staff to review the PR.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,71 @@
...
+## Phase I. (ongoing)
+
+<details>
+<summary>Evaluation DAO (30%)</summary>
+
+### Evaluation DAO
+
+This complex challenge seeks your skills in DAO development and implementation and is one of the most important challenges of phase one. The Evaluation DAO will ensure that contributions in Game of Realms and the Gno.land platform are fairly rewarded.
+
+- [x] Clarifying this issue — [100% completed]
```
Comment (ajnavarro) [inline README.md:22]: shall we add the links to the issues to have more context? If the issue contains its own checklist, the process will be updated automatically.

**Review by ajnavarro — COMMENTED** [review]
good approach! I would say to add issue links instead of just plain text to avoid having outdated data here (it will be updated automatically with the issue state, open/closed/%)

File: README.md (hunk 1 above)
Comment (ajnavarro) [reply README.md:22]:
```suggestion
- [x] https://github.com/gnolang/gno/issues/407
```

Comment (roethke) [reply README.md:22]: I really like this suggestion and to make things automated/less error-prone/fewer updates needed; this is great. I'll speak with the Marketing team to get their thoughts.

**Review by moul — APPROVED** [review]

**Comment by moul** [comment]
Although there's room for enhancement, let's proceed with merging. Our immediate focus should be on updating this content and the website. Ultimately, our goal is to transition to complete on-chain management.

**Pull request merged**